README.md

```diff
@@ -501,6 +501,16 @@ This behavior applies to uncaught exceptions, not direct calls to `Rollbar.error
 Rollbar.error(exception, :use_exception_level_filters => true)
 ```
 
+### Advanced Usage
```

Comment:
"Advanced usage" is sorta vague - how about a heading that describes what this does, maybe "Dynamic levels" ?